src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -510,29 +541,33 @@ def main():
         print(f"{wtus_len} assemblies in {data_dir} are ready to upload to workspace {workspace_id}")
 
         start = time.time()
-        failed_names = _upload_assembly_files_in_parallel(
+        uploaded_count = _upload_assembly_files_in_parallel(
             conf,
             env,
             workspace_id,
             upload_dir,
             wait_to_upload_assemblies,
+            batch_size,
         )
-        
-        assembly_count = wtus_len - len(failed_names)
+
         upload_time = (time.time() - start) / 60
-        assy_per_min = assembly_count / upload_time
+        assy_per_min = uploaded_count / upload_time
 
-        print(f"\n{workers} workers took {upload_time:.2f} minutes to upload {assembly_count} assemblies, "
+        print(f"\ntook {upload_time:.2f} minutes to upload {uploaded_count} assemblies, "
               f"averaging {assy_per_min:.2f} assemblies per minute")
 
-        if failed_names:
-            raise ValueError(f"\nFailed to upload {failed_names}")
-
     finally:
         # stop callback server if it is on
         if conf:
             conf.stop_callback_server()
 
+        # print out container logs
```

Comment:
This code should live in the Conf class. Is there any reason it shouldn't just run when the stop method is called?

Reply:
Are you proposing this code should live in the Conf class and on its own(not inside stop_callback_server fun) or logs should be out as soon as they are ready?

Reply:
> Are you proposing this code should live in the Conf class and on its own(not inside stop_callback_server fun)

It could either be a part of the stop function if the logs should be dumped whenever the server stops, or it could be a new function (which could be called by the stop function)

Reply:
Updated!

Reply:
:+1: